web/client/plugins/userExtensions/withPluginsDefinition.js

```diff
@@ -0,0 +1,33 @@
+import { compose, mapPropsStream, withPropsOnChange } from "recompose";
+import { find, pick } from 'lodash';
+
+import Rx from 'rxjs';
+const loadingState = require('../../components/misc/enhancers/loadingState');
+
+import getPluginsConfig from '../../observables/config/getPluginsConfig';
+/**
+ * Adds to user extensions the plugins definition and maps the properties
+ * needed (title, description and glyph) to correctly set labels and symbology,
+ * accordingly with the context manager.
+ */
+export default (pluginsConfigURL) => compose(
+    mapPropsStream(props$ =>
```

Comment:
honestly, the less we use mapPropsStream the better it is, but that's my opinion

Reply:
There are many of them in MapStore. we can stop using them after a discussion, but is not a decision to take during a PR review